### ISSUE TITLE
produce details page

### ISSUE THREAD

**Issue by carolinasno** [carolinasno/inseason#1]
The goal is to have the produce details page show the details of the fruit/vegetable the user clicks on from the produce list page. The function I'm currently using keeps showing the last object from the produce json file and I'm not sure where I'm going wrong here. Please help!

**Comment by ajhofferber** [comment]
Hey Carolina, 

Did you get a chance to work with someone on this issue yesterday?

**Comment by carolinasno** [comment]
No, but I'm going to try to work with ngRoutes again on this first. I found a guide that will help compile the jade files to html, which hopefully helps. Could I slack you afterwards? I also have a few questions about map markers.

**Comment by ajhofferber** [comment]
yeah for sure! just let me know

**Comment by carolinasno** [comment]
Awesome. Thanks!
